collect_app/src/main/AndroidManifest.xml

```diff
@@ -82,6 +82,7 @@ the specific language governing permissions and limitations under the License.
         android:installLocation="auto"
         android:label="@string/collect_app_name"
         android:largeHeap="true"
+        android:enableOnBackInvokedCallback="true"
```

Comment:
I don't think we should actually opt-in yet. Even opting in, you don't see the new behaviour unless predictive back is enabled in Developer Settings on the device. That said, they might change that in a later Android release and we wouldn't want a problem in predictive back catching us by surprise. 

I think we also would want to do some more work to support predictive back throughout the app so the experience is a bit more consistent before opting in, but before that I'd like to see this feature actually be enabled for general Android users. Being really cynical, I wouldn't be surprised if this feature is never fully released. Flows like our save/discard dialog just don't fit into that model, and Google haven't been able to remove 3/2 button based navigation (it's still available in Android 14) which I think will be precursor.

Reply:
Ah yeah e don't need it now. Removed.